tests/migration/migration.go

```diff
@@ -2938,6 +2941,65 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 		})
 	})
 
+	Context("[Serial]Cluster-Wide CPU Feature Disablement using cpu-model-to-features-to-disable ConfigMap", Serial, func() {
+		var cpuFeaturesToDisbale = "aes"
+		var cpuModelsThatRequireAES = []string{"Broadwell-IBRS", "Broadwell-noTSX-IBRS", "Broadwell-noTSX", "Broadwell",
+			"Cascadelake-Server-noTSX", "Cascadelake-Server", "Cooperlake", "EPYC-IBPB", "EPYC-Milan", "EPYC-Rome",
+			"EPYC", "Haswell-IBRS", "Haswell-noTSX-IBRS", "Haswell-noTSX", "Haswell", "Icelake-Client-noTSX",
+			"Icelake-Client", "Icelake-Server-noTSX", "Icelake-Server", "IvyBridge-IBRS", "IvyBridge", "Opteron_G4",
+			"Opteron_G5", "SandyBridge-IBRS", "SandyBridge", "Skylake-Client-IBRS", "Skylake-Client-noTSX-IBRS",
+			"Skylake-Client", "Skylake-Server-IBRS", "Skylake-Server-noTSX-IBRS", "Skylake-Server", "Snowridge",
+			"Westmere-IBRS", "Westmere",
+		}
+		var chosenModel string
+		BeforeEach(func() {
+			nodes := libnode.GetAllSchedulableNodes(virtClient)
+			for _, model := range cpuModelsThatRequireAES {
+				if LabelExistOnAtLeastTwoNodes(nodes.Items, v1.CPUModelLabel+model) {
+
+					cm := &k8sv1.ConfigMap{
+						ObjectMeta: metav1.ObjectMeta{
+							Name: util2.CpuModelDisableFeaturesConfigMapName,
+						},
+						Data: map[string]string{model: cpuFeaturesToDisbale},
+					}
+					_, err := virtClient.CoreV1().ConfigMaps(flags.KubeVirtInstallNamespace).Create(context.Background(), cm, metav1.CreateOptions{})
+					Expect(err).ToNot(HaveOccurred())
+
+					chosenModel = model
+					return
+				}
+			}
+			Skip("cannot run this test when CPU Models with aes are not supported")
+		})
+		It("Feature Disablement should be consistent on live migration even after cpu-model-to-features-to-disable is deleted", func() {
+			vmi := libvmi.NewGuestless(
+				libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+				libvmi.WithNetwork(v1.DefaultPodNetwork()),
+				libvmi.WithCPUModel(chosenModel),
+			)
+			vmi = tests.RunVMIAndExpectLaunch(vmi, 240)
+
+			By("Trying to migrate the VirtualMachineInstance")
```

Comment:
Please move this in the line before you are going to create the migration

Reply:
Done